scanning/disa-alignment/shared.py

```diff
@@ -0,0 +1,86 @@
+import re
+
+from lib import results, versions
+
+
+profile = 'stig'
+profile_full = f'xccdf_org.ssgproject.content_profile_{profile}'
+
+# old RHEL-7 oscap mixes errors into --progress rule names without a newline
+redir = '2>&1' if versions.oscap >= 1.3 else ''
+# RHEL-7 HTML report doesn't contain OVAL findings by default
+oval_results = '' if versions.oscap >= 1.3 else '--results results.xml --oval-results'
+
+shared_cmd = ['oscap', 'xccdf', 'eval', '--progress', oval_results]
+
+
+def content_scan(host, ds):
+    '''
+    Scan machine and prepare ARF results for STIG Viewer.
+    Return HTML report and ARF file.
+    '''
```

Comment:
We use `"` (double quotes) for all other docstrings in Contest, per https://github.com/RHSecurityCompliance/contest/blob/main/STYLE.md#quotes .